docs/content/users/usage/architecture.md

```diff
@@ -102,6 +102,27 @@ Files beginning with `.` are hidden because they shouldn’t be fiddled with; mo
 
 There’s only one global `.ddev` directory, which lives in your home directory: `~/.ddev` (`$HOME/.ddev`).
 
+!!!tip "Where is my global `.ddev` config?"
+    Use `ddev config global` (look at `global-config-path`) to check which location is currently used for the `.ddev` global directory.
```

Comment:
Now I see why you have the global config element, but it seems it's not config but reporting?

Reply:
Yes, that's right, I thought it would be nice to have this when users send us `ddev debug test`.